src/node_process_methods.cc

```diff
@@ -278,6 +278,29 @@ void GetActiveHandles(const FunctionCallbackInfo<Value>& args) {
       Array::New(env->isolate(), handle_v.data(), handle_v.size()));
 }
 
+static void ResourcesUsage(const FunctionCallbackInfo<Value>& args) {
+  uv_rusage_t rusage;
+  int err = uv_getrusage(&rusage);
+  if (err) {
+    // On error, return the strerror version of the error code.
+    Local<String> errmsg = OneByteString(args.GetIsolate(), uv_strerror(err));
```

Comment:
Can we use `Environment::ThrowUVException` instead of stringifying the error and returning it? See e.g. the implementation of `Cwd()` in this file for how to use that. In that case, we should also be able to drop the new error code here (which would imo be a good thing, because error codes should reflect the type of error that happened, and not its origin).

Reply:
:+1: